vllm/envs.py

```diff
@@ -518,6 +519,10 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
     "VLLM_USE_V1":
     lambda: bool(int(os.getenv("VLLM_USE_V1", "0"))),
 
+    # If set, use the V1 code path.
```

Comment:
note: this is temporary to merge this PR. This will be removed shortly and will require fighting the CI